apps/jobs/models.py

```diff
@@ -106,6 +106,7 @@ class Submission(TimeStampedModel):
     method_description = models.TextField(blank=True, default="")
     publication_url = models.CharField(max_length=1000, default="", blank=True)
     project_url = models.CharField(max_length=1000, default="", blank=True)
+    baseline_submission = models.BooleanField(default=False)
```

Comment:
Can we please modify the field name to `is_baseline`. Since we're adding it in Submission model hence it'll be implicit that it is a submission.